app/views/users/show.html.erb

```diff
@@ -1,5 +1,6 @@
 <section class="user-section">
-  <h2><%= "Name: #{@user.name}" %></h2>
+  <h2><%= "Name: #{@user.name}" %>     </h2>
+   <%= button_to "Invite to Friendship",friendships_path(user_id:@user)%>
   <article class="timeline">
```

Comment:
I shouldn't be able to see other people's posts UNLESS I'm friends with them. Currently, I can see everyone's posts. See [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/friends_timeline.png).